packages/cacti-plugin-consortium-static/src/main/typescript/policy-model/polity-item.ts

```diff
@@ -0,0 +1,47 @@
+import { IPolicy } from "./i-policy";
```

Comment:
Please elaborate (can be on the README) on the relationship between conditions, groups, rules and items (this is already partially done, but would be good to make everything explicit)